9.4/paper.bib

```diff
@@ -491,7 +491,8 @@ @Misc{cpp14
   howpublished = {\url{https://www.iso.org/standard/64029.html}},
   year =         2014}
 
-@Article{trilinos,
+See: https://trilinos.github.io/cite.html
```

Comment:
The projects asks to cite the website, so I think we should keep both entries.

Reply:
If we just add the website for the Trilinos project, I fell it's unfair compared to the other libraries. We should be consistent.

I tried to find the best of both worlds with this PR: I added the website as a note to each publication we reference for every library (see line 505 in this patch for Trilinos). Looks like this in the paper: ![image](https://user-images.githubusercontent.com/18285973/175827025-13f1c94c-b46d-409f-b3dc-d5cd00e292db.png)

What do you all think about this approach?

Reply:
My point is that I am okay with two citations per project if the project asks to be cited that way. Adding a project url to a citation is a bit weird on my opinion and I would rather use a footnote or a bib entry.